use-cases/src/Contracts/CrowdFunding.hs

```diff
@@ -0,0 +1,119 @@
+{-# LANGUAGE TemplateHaskell #-}
+
+-- This is the fully parallel version that collects all contributions
+-- in a single transaction. This is, of course, limited by the maximum
+-- number of inputs a transaction can have.
+
+module Contracts.CrowdFunding (contribute, collect, refund)
+where
+
+import           Control.Monad (void)
+import           Plutus
+{-# ANN module "HLint: ignore Avoid restricted function" #-}

```

Comment:
prefer putting this in the hlint config file, I think, so it's obvious where we use this